packages/plaid-funding-source/.env.local.example

```diff
@@ -1,11 +0,0 @@
-# Dwolla Environment and OAuth Keys
```

Comment:
Looks like this file was removed as part of the commit. Can this be brought back?